packages/kinvey-nativescript-sdk/package.json

```diff
@@ -36,15 +40,26 @@
     "push.d.ts",
     "push.ios.js",
     "push.ios.js.map",
+    "lib/before-prepare.js",
```

Comment:
The file's actual name is before-checkForChanges.js

Reply:
Fixed